src/main/java/com/uber/cadence/worker/Worker.java

```diff
@@ -545,7 +545,9 @@ public void start() {
     }
 
     /**
-     * Shuts down Poller used for sticky workflows and all workers created by this factory.
+     * Shuts down Poller used for sticky workflows and all workers created by this factory. Shutdown
+     * will be called on each worker created by this factory with the given timeout. If the timeout
+     * is exceed the method will thrown an InterruptedException.
```

Comment:
throwing Interrupted on timeout is not a standard Java practice. In case of shutdown returning without error (may be logging a warning) is a preferred option

Reply:
resovled